app/models/participant_declaration.rb

```diff
@@ -1,7 +1,7 @@
 # frozen_string_literal: true
 
 class ParticipantDeclaration < ApplicationRecord
-  self.ignored_columns = %w[statement_type]
+  self.ignored_columns = %w[statement_type statement_id]
```

Comment:
why is this necessary? the column will be removed so there is nothing to ignore. 🤔

Reply:
the strong migration thing complains at me. if it weren't there personally I wouldn't add it